README.md

```diff
@@ -78,8 +87,13 @@ If you just want to run the system, see "Run the Code" below.
    Note that this script is just a convenience to install system-wide dependencies we expect.
    As a result, it uses the system package manager, requires `sudo`, and should only be run **once**.
    ```console
-   # ./scripts/install-build-tools.sh
+   $ sudo ./scripts/install-build-tools.sh
```

Comment:
This should be restored to # to be consistent with our other docs (further it makes the below note unnecessary)